futures-util/src/io/mod.rs

```diff
@@ -312,7 +312,7 @@ pub trait AsyncReadExt: AsyncRead {
     /// use futures::io::{self, AsyncReadExt, Cursor};
     ///
     /// // Note that for `Cursor` the read and write halves share a single
-    /// // seek position. This may or may not be true for other types that
+    /// // seek position. This may or may not be `true` for other types that
```

Comment:
This isn't referring to `true` the value in the language, but the truth of a statement (whether read and write halves share a single seek position).